BackEndFlask/setupEnv.py

```diff
@@ -24,6 +24,46 @@ def main():
             print("[Server] attempting to run pip install requirements failed...")
             print("[Server] exiting...")
             os.abort()
+     try:
+        print("\n[Server] attempting to install Homebrew using setupHomebrew.sh...\n")
+        if(os.system("chmod 755 setupHomebrew.sh") != 0):
+            raise Exception
+        if(os.system("./setupHomebrew.sh") != 0):
+            raise Exception
+        if(os.system("brew --version") != 0):
+            raise Exception
+        if(os.system("brew install redis") != 0):
+            raise Exception
+        if(os.system("brew services start redis") != 0):
+            raise Exception
+        if(os.system("brew services info redis") != 0):
+            raise Exception
+        time.sleep(sleepTime)
```

Comment:
Why is time.sleep() called? Is it to give the system time to do something?

Reply:
During development, time.sleep() was used to give enough time to view errors easier in the terminal. I have set the sleepTime to 0.0.